client/ayon_zbrush/api/plugin.py

```diff
@@ -84,6 +84,7 @@ def get_dynamic_data(self, *args, **kwargs):
         output = {}
         if asset_name:
             output["asset"] = asset_name
+            output["folderPath"] = asset_name
```

Comment:
Shouldn't this technically be `folder[path]` and `folder[type]` or what are we matching this dynamic data with?

Reply:
Already updated in [dc5fe1c](https://github.com/ynput/ayon-zbrush/pull/22/commits/dc5fe1c5db4375b4a1455be9ebef8d2414cebe99) and this similar thing need to adjusted in SpeedTree too.